app/model/editions/templates/TheDummyEdition.scala

```diff
@@ -14,6 +14,7 @@ object TheDummyEdition extends InternalEdition {
   override val header = Header("The Dummy", Some("Edition"))
   override val notificationUTCOffset = 3
   override val topic = "dm"
+  override val locale = Some("dm_DM")
```

Comment:
I'm concerned this might prevent us viewing the Dummy Edition on devices.  May I suggest en_GB instead?